explainaboard/processors/processor.py

```diff
@@ -228,6 +227,37 @@ def _init_customized_features(self, metadata: dict):
                 else:
                     raise NotImplementedError
 
+    def _get_max_min_value(
```

Comment:
This seems to not be necessary, let's just take the min and max in the bucketing functions.

Reply:
Similar to the above reply.  I tend to keep it.